frontend/src/app/navigation/navigation.component.ts

```diff
@@ -28,4 +45,11 @@ export class NavigationComponent implements OnInit {
     }
   }
 
+  goToEventPage(id: string){
+    //DO NOT CHANGE THIS
```

Comment:
Wouldn't it be better if we kept the comments in a single block above the line, instead of one above and one below?